StreamerBotPlugin/Properties/AssemblyInfo.cs

```diff
@@ -30,4 +30,4 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("1.0.2")]
+[assembly: AssemblyVersion("1.0.2.*")]
```

Comment:
This could be problematic due to my bump/changelog/release script. I use [SemVer](https://semver.org/) versioning.